webapp/channels/src/components/admin_console/admin_definition.tsx

```diff
@@ -2442,50 +2444,145 @@ const AdminDefinition: AdminDefinitionType = {
+                                isHidden: it.not(it.all(
+                                    it.stateIsTrue('AutoTranslationSettings.Enable'),
+                                    it.stateEquals('AutoTranslationSettings.Provider', 'libretranslate'),
+                                )),
+                                isDisabled: it.not(it.userHasReadPermissionOnResource(RESOURCE_KEYS.SITE.LOCALIZATION)),
+                                label: defineMessage({id: 'admin.general.localization.autoTranslateProviderLibreTranslateURLTitle', defaultMessage: 'LibreTranslate API Endpoint'}),
+                                placeholder: defineMessage({id: 'admin.general.localization.autoTranslateProviderLibreTranslateURLExample', defaultMessage: 'e.g.: "https://libretranslate.yourdomain.com"'}),
+                            },
+                            {
+                                type: 'text',
+                                key: 'AutoTranslationSettings.LibreTranslate.APIKey',
+                                isHidden: it.not(it.all(
+                                    it.stateIsTrue('AutoTranslationSettings.Enable'),
+                                    it.stateEquals('AutoTranslationSettings.Provider', 'libretranslate'),
+                                )),
+                                isDisabled: it.not(it.userHasReadPermissionOnResource(RESOURCE_KEYS.SITE.LOCALIZATION)),
+                                label: defineMessage({id: 'admin.general.localization.autoTranslateProviderLibreTranslateAPIKeyTitle', defaultMessage: 'LibreTranslate API Key'}),
+                                placeholder: defineMessage({id: 'admin.general.localization.autoTranslateProviderLibreTranslateAPIKeyExample', defaultMessage: 'Enter LibreTranslate API Key'}),
+                                help_text: defineMessage({id: 'admin.general.localization.autoTranslateProviderLibreTranslateAPIKeyDescription', defaultMessage: 'If your LibreTranslate server requires an API key, enter it here. Otherwise, leave this field blank. View <link>LibreTranslate docs</link> for API Key management.'}),
+                                help_text_markdown: false,
+                                help_text_values: {
+                                    link: (msg: string) => (
+                                        <ExternalLink
+                                            location='admin_console'
+                                            href='https://docs.libretranslate.com/guides/manage_api_keys/'
+                                        >
+                                            {msg}
+                                        </ExternalLink>
+                                    ),
+                                },
+                            },
+                            {
+                                type: 'custom',
+                                key: 'AutoTranslateInfo',
+                                component: AutoTranslateInfo,
```

Comment:
We should already have a notice component like this here: webapp/channels/src/components/section_notice